.pre-commit-config.yaml

```diff
@@ -68,3 +65,9 @@ repos:
           # stylelint itself needs to be here when using additional_dependencies.
           - stylelint@16.5.0
           - stylelint-config-standard-scss@13.1.0
+
+  - repo: "https://github.com/pre-commit/pre-commit-hooks"
+    rev: v5.0.0
+    hooks:
+      - id: trailing-whitespace
+      - id: end-of-file-fixer
```

Comment:
I thought ruff would fix both of these?  Or is this here to handle file types that ruff doesn't touch?

Reply:
Yes ruff won't touch md, rst, and our translation files which is how I noticed we could do with adding these.